Doc/whatsnew/3.15.rst

```diff
@@ -536,6 +528,14 @@ tarfile
   (Contributed by Christoph Walcher in :gh:`57911`.)
 
 
+timeit
+------
+
+* Error tracebacks are now colourised by default. This can be controlled by
+  :ref:`environment variables <using-on-controlling-color>`.
+  (Contributed by Yi Hong in :gh:`139374`.)
```

Comment:
```suggestion
* The command-line interface now colorize error tracebacks
  by default. This can be controlled by
  :ref:`environment variables <using-on-controlling-color>`.
  (Contributed by Yi Hong in :gh:`139374`.)
```